pyart/io/nexrad_level2.py

```diff
@@ -426,7 +434,10 @@ def get_target_angles(self, scans=None):
         if scans is None:
             scans = range(self.nscans)
         if self._msg_type == '31':
-            cut_parameters = self.vcp['cut_parameters']
+            if self.vcp is not None:
+                cut_parameters = self.vcp['cut_parameters']
+            else:
```

Comment:
@scollis I would possibly move the warning here instead. It seems to only effect the fixed angle data, and pytest throws warnings on the testing nexrad file, but if the warning is moved here, it still catches and does what it is suppose to do, but doesn't throw the warning in the testing suite.

Reply:
Actually I disagree. I want it when the MSG5 is found to me missing in case anyone adds anything else.

Reply:
Ah makes sense, yeah disregard.